ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/compact-3node-cluster/cucushift-installer-rehearse-vsphere-ipi-compact-3node-cluster-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-ipi-compact-3node-cluster
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-compact-3node-cluster-provision
+    post:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-compact-3node-cluster-deprovision
```

Comment:
I guess we can use `cucushift-installer-rehearse-vsphere-ipi-deprovision ` chain directly, so no need to add the `cucushift-installer-rehearse-vsphere-ipi-compact-3node-cluster-deprovision` chain.

Reply:
ok， got it.